contract/src/constants.rs

```diff
@@ -70,6 +72,10 @@ pub const PAGE_LIMIT: &str = "page_limit";
 pub const UNMATCHED_HASH_COUNT: &str = "unmatched_hash_count";
 pub const MIGRATION_FLAG: &str = "migration_flag";
 pub const REPORTING_MODE: &str = "reporting_mode";
+pub const EVENTS: &str = "events";
+pub const EVENT_ID_TRACKER: &str = "id_tracker";
```

Comment:
```suggestion
```

Does not seem to be used anywhere except `init`

Reply:
Removed unneeded constant